server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/create/CreateSnapshotRequest.java

```diff
@@ -72,7 +73,9 @@
 
     private String[] indices = EMPTY_ARRAY;
 
-    private IndicesOptions indicesOptions = IndicesOptions.strictExpand();
+    private IndicesOptions indicesOptions = IndicesOptions.strictExpandHidden();
+
+    private boolean defaultExpandWildcards = true;
```

Comment:
This seems kind of wrong to me. We're adding something to the request here only so that we can log the deprecation warning accordingly?

Reply:
Yeah, that felt wrong when I was writing it, and after sleeping on it I'm not sure what I was thinking. I've reverted the commit that adds the deprecation warning (and this field in service of it) and will do some more work on that part and break it out into another PR so we can get this into patch releases as appropriate quickly.